packages/glibc223.rb

```diff
@@ -3,17 +3,142 @@
 class Glibc223 < Package
   description 'GNU C Library'
   homepage 'https://www.gnu.org/software/libc/'
-  version '2.23-2'
+  version '2.23-3'
+  source_url 'https://ftp.gnu.org/gnu/glibc/glibc-2.23.tar.xz'
```

Comment:
Can we use `https://ftpmirror.gnu.org/glibc/glibc-2.23.tar.xz` instead?  This should localize the downloads better.